### PR TITLE
fix(nav) Navigation links

### LINKED ISSUE #2020: Navigation links are invalid
All navigation links from https://github.com/webpack/webpack.js.org/blob/master/src/components/Navigation/Links.json get a `/` prepended in https://github.com/webpack/webpack.js.org/blob/master/src/components/Navigation/Navigation.jsx#L33 and https://github.com/webpack/webpack.js.org/blob/master/src/components/Navigation/Navigation.jsx#L87

Since some Urls in the json are protocol relative links (eg. `//medium.com/webpack`), the resulting href in the anchors have three slashes in the beginning. This is not a valid url.

image

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Dont modify external urls in nav bars

Fixes #2020

### CODE REVIEW

**Comment by montogeek** [comment]
This is fine for now, but we need to rethink our URL parsing and formatting in `rebuild` branch, right now we are parsing them from markdown and transforming them later again.

**Comment by EugeneHlushko** [comment]
Sounds good, by the way can you remind me where is the main discussion going on for `rebuild` branch? Sean hadnt invited me to slack

**Comment by montogeek** [comment]
@TheLarkInn Please invite @EugeneHlushko to Slack

File: src/components/Navigation/Navigation.jsx [hunk 1]
```diff
@@ -170,4 +170,14 @@ export default class Navigation extends React.Component {
 
     container.classList.add('navigation--search-mode');
   }
+
+  /**
+   * Builds an url prefixed and suffixed with slashes for internals
+   *
+   * @param  {object} link - An object describing the `link`
+   * @return {string}
```
Comment (dhruvdutt) [inline src/components/Navigation/Navigation.jsx]: Can you add the description of return param?

**Comment by jeremenichelli** [comment]
@EugeneHlushko is this really solving an issue? I just clicked the blog url and it works just fine.

**Comment by jeremenichelli** [comment]
If it's not a big issue, yeah I prefer to not merge and cause less friction on rebuild rebase. If it's definitely an issue we should to provide a stable experience now.

**Comment by EugeneHlushko** [comment]
closing this to not create overhead on rebuild rebase

**Pull request closed without merging**